data/categoryRepository.go

```diff
@@ -125,7 +124,6 @@ LEFT JOIN measurement_series ON measurement_series.measurement_id = data_list_me
 LEFT JOIN series ON series.id = measurement_series.series_id
 LEFT JOIN feature_toggles ON feature_toggles.name = 'filter_by_quarantine'
```

Comment:
Would it be good to get rid of the join as well, for good luck? Doesn't look like the join is being used elsewhere in query (even if it doesn't hurt). Also... there are other queries that do this kinda join to find 'filter_by_quarantine'. Just checking that the two instances you removed are the only relevant ones, and the ones that remain are ok.